tests/lib/config/config-initializer.js

```diff
@@ -66,16 +66,12 @@ describe("configInitializer", function() {
         sh.mkdir("-p", fixtureDir);
         sh.cp("-r", "./tests/fixtures/config-initializer/.", fixtureDir);
         fixtureDir = fs.realpathSync(fixtureDir);
+        cwd = process.cwd();
+        process.chdir(fixtureDir);
```

Comment:
I was told to try to avoid `process.chdir` in our tests, as it can cause some hard-to-track-down bugs.  Couldn't we leave the npm check stubbed, and add tests to the npm-util?

Totally my bad for not considering an empty package.json.  Thanks for finding/fixing.